integrations/sbt-bloop/src/main/scala/bloop/integrations/sbt/SbtBloop.scala

```diff
@@ -60,6 +60,7 @@ object PluginImplementation {
   // We create build setting proxies to global settings so that we get autocompletion (sbt bug)
   val buildSettings: Seq[Def.Setting[_]] = List(
     BloopKeys.bloopInstall := BloopKeys.bloopInstall.in(Global).value,
+    // Bloop users: Do NEVER override this setting as a user if you want it to work
```

Comment:
This should probably be in the documentation, no?

Reply:
We don't have docs for this yet. Do you want me to add them?

Reply:
Just added them.